backend/external-events/events.go

```diff
@@ -0,0 +1,121 @@
+package externalevents
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/ansel1/merry/v2"
+)
+
+// Based on https://github.com/bcc-code/brunstadtv/blob/ad2e2942eaff5530830fb6a3a3ed5ee449a71cfd/backend/cmd/events/Readme.md
+/*
+{
+	"timestamp": "2009-11-11T08:00:00+09:00", // RFC3339 formatted
+	"source": "some-system", // Arbitrary string identifying the source of the message, Informative
+	"type": "event.happened", // Arbitrary string, identifying the format of the data sent
+	"data": {"A": "B", "C": 99 }, // Arbitrary json object containing the actual data
+}
+*/
+
+// Event is the base event wihth data represented as raw json
+type Event struct {
```

Comment:
Just a thought.

Maybe "hide" BibleData, SongData, etc behind GetBibleData / GetSongData functions?
We have to check the type whenever we want to retrieve the event data, so might as well wait with the deserialization until we need it, or? That way, we don't need to make pointers, as we know they are not nil.

Reply:
I'll actually wait with this as I don't know how this will be used internally yet.

Reply:
Or, no I'll hide them, Makes it easier to not serialize to json.